### ISSUE TITLE
Metadata creation fail

### ISSUE THREAD

**Issue by ChristopherTracey** [PNHP/Regional_SDM#25]
Keeping getting a fatal error on the metadata creation step that seems to be a result of missing citations in the rnw.  See the folllowing from the log file:

```
LaTeX Warning: Citation `breiman2001' on page 1 undefined on input line 98.
LaTeX Warning: Citation `iverson2004' on page 1 undefined on input line 98.
LaTeX Warning: Citation `liaw2002' on page 1 undefined on input line 99.
LaTeX Warning: Citation `r' on page 1 undefined on input line 99.
LaTeX Warning: Citation `nhd' on page 1 undefined on input line 100.
LaTeX Warning: Citation `fielding1997' on page 1 undefined on input line 102.
LaTeX Warning: Citation `fielding2002' on page 1 undefined on input line 102.
LaTeX Warning: Citation `pearson2007' on page 1 undefined on input line 102.
```

**Comment by dnbucklin** [comment]
should be fixed with 0ff9e2835f607ae314ee2cbad4485b1bf35237b5 ... this was due to a pretty troublesome bug that resulted in species codes of `[speciescode]_att.csv` (this also affected the model_run_name), and the cascading effect was that it caused metadata to error because it couldn't find the messed-up species code in the database.

Unfortunately I had introduced this in 1998662d5940d1211bc26c737574bf20eda7aa2e by using `.dbf` instead of `.csv` in a `gsub` call.

**Comment by ChristopherTracey** [comment]
Cool. I saw the `[speciescode]_att.csv` popping up in the file names, and was wondering if that was the cause.
